nuxeo-services/login/nuxeo-platform-login-openid/src/main/java/org/nuxeo/ecm/platform/oauth2/openid/OpenIDConnectProvider.java

```diff
@@ -89,7 +96,8 @@
 
     public OpenIDConnectProvider(OAuth2ServiceProvider oauth2Provider, String accessTokenKey, String userInfoURL,
             Class<? extends OpenIDUserInfo> openIdUserInfoClass, String icon, boolean enabled,
-            RedirectUriResolver redirectUriResolver, Class<? extends UserResolver> userResolverClass, String userMapper) {
+            RedirectUriResolver redirectUriResolver, Class<? extends UserResolver> userResolverClass,
+            String userMapper) {
```

Comment:
Please undo formatting, this should be a separate commit

Reply:
Done